lib/galaxy/jobs/__init__.py

```diff
@@ -1743,7 +1743,14 @@ def _change_ownership( self, username, gid ):
         job = self.get_job()
         # FIXME: hardcoded path
         external_chown_script = self.get_destination_configuration("external_chown_script", None)
-        cmd = [ '/usr/bin/sudo', '-E', external_chown_script, self.working_directory, username, str( gid ) ]
+
+        cmd = [ '/usr/bin/sudo', '-E', ]
+        for v in ["PATH", "LD_LIBRARY_PATH", "PKG_CONFIG_PATH"]:
```

Comment:
Would it be an option to call 

`/path/to/python external_chown_script.py ... ` 

instead of 

`external_chown_script.py ... ` (using the shellbang) 

Then we could just use the python of galaxy's virtual environment if this path is accessible from the code. But this still leaves the problem of the library path (I never understood why the python binary but not the shared object is copied to the venv).